src/encoder.rs

```diff
@@ -197,8 +197,7 @@ pub struct Sequence {
   pub operating_point_idc: [u16; MAX_NUM_OPERATING_POINTS],
   pub display_model_info_present_flag: bool,
   pub decoder_model_info_present_flag: bool,
-  /// minor, major
-  pub level: [[usize; 2]; MAX_NUM_OPERATING_POINTS],
+  pub level_idx: [usize; MAX_NUM_OPERATING_POINTS],
```

Comment:
Wasn't this changed to u8 in the previous iteration? It would spare you from casting it when writing the value.

Reply:
last iteration didn't save level_idx here, u8 means I have to cast in other locations in the next patch but I'll go with u8 to save the ram